addendum.dict

```diff
@@ -298,24 +306,28 @@ Pickering	𐑐𐑦𐑒𐑼𐑦𐑙
 Pilbeam	𐑐𐑦𐑤𐑚𐑰𐑥
 Pinkerton	𐑐𐑦𐑙𐑒𐑼𐑑𐑩𐑯
 Plainpalais	𐑐𐑤𐑧𐑯𐑐𐑨𐑤𐑱
+PlayStation 𐑐𐑤𐑱𐑕𐑑𐑱𐑖𐑩𐑯
 Plutarch	𐑐𐑤𐑵𐑑𐑸𐑒
 Polee	𐑐𐑴𐑤𐑦
 preggo	𐑐𐑮𐑧𐑜𐑴
 Procter	𐑐𐑮𐑪𐑒𐑑𐑼
 Prometheus	𐑐𐑮𐑴𐑥𐑰𐑔𐑾𐑕
 Putin	𐑐𐑵𐑑𐑦𐑯
 Qanon	𐑒𐑿-𐑩𐑯𐑪𐑯
+Quickscript 𐑒𐑢𐑦𐑒𐑕𐑒𐑮𐑦𐑐𐑑
 Radlkofer	𐑮𐑭𐑛𐑩𐑤𐑒𐑪𐑐𐑼
 Rafael	𐑮𐑨𐑓𐑱𐑩𐑤
 Raines	𐑮𐑱𐑯𐑟
 Ramsay	𐑮𐑨𐑥𐑟𐑦
+Readspel 𐑮𐑰𐑛𐑕𐑐𐑧𐑤
 Reddit	𐑮𐑧𐑛𐑦𐑑
 Reggie	𐑮𐑧𐑡𐑦
 Reigelheimer	𐑮𐑲𐑜𐑩𐑤𐑣𐑲𐑥𐑼
 Reuss	𐑮𐑶𐑕
 Rhone	𐑮𐑴𐑯
 Rieder	𐑮𐑰𐑛𐑼
 Rittenhouse	𐑮𐑦𐑑𐑩𐑯𐑣𐑬𐑕
+Rockafellar 𐑮𐑪𐑒𐑭𐑓𐑧𐑻
```

Comment:
Should be 𐑮𐑪𐑒𐑩𐑓𐑧𐑤𐑼 to comply with principles 14 and 8/9 at https://www.shavian.info/spelling/. Also the Latin has a a couple of typos I think, i.e. it's usually Rockefeller (see https://en.wikipedia.org/wiki/John_D._Rockefeller).